integration-cli/docker_cli_pull_test.go

```diff
@@ -55,6 +55,8 @@ func TestPullImageWithAliases(t *testing.T) {
 
 // pulling library/hello-world should show verified message
 func TestPullVerified(t *testing.T) {
+	t.Skip("Skipping hub dependent test")
```

Comment:
Does it need a FIXME / TODO (and a related issue `#`), so it's not forgotten to re-enable/rewrite it?